src/components/search-bar/search-bar.scss

```diff
@@ -6,54 +6,98 @@
 $height: 40px;
 $font-size: 20px;
 
-.search-bar-wrapper {
-  display: inline-flex;
-  justify-content: space-between;
+.c-search-bar-wrapper {
+  position: relative;
   border: 1px solid $light-blue;
   width: 100%;
   height: $height;
   background-color: $black;
 
-  .search-icon-container {
-    @include flex-center;
-    width: $height;
-    height: $height;
+  .c-search-bar-content {
+    display: inline-flex;
+    justify-content: space-between;
+    width: 100%;
 
-    &.has-text {
-      border-right: 1px solid $light-blue;
-      background-color: $blue;
+    .c-search-icon-container {
```

Comment:
I like your naming convention for components. I'll start doing that too, and I'll go back and update the other classes as well

Reply:
Cool! I ran into a conflict in class namespaces and decided to add c- to all classes in our components to avoid that. :P